faq.md

```diff
@@ -81,45 +77,14 @@ By March 25 or when we sell out, whichever comes first! We can't wait to see you
-
-##### Which airport should I fly into?
-
-The best airport is probably San Francisco International Airport (SFO), but if you are coming early we suggest coordinating with your host—they may live closer to a different airport (such as OAK). Both are within an hour's drive to the venue.  SJC is about an hour from San Francisco/Oakland, so please plan to rent a car if you fly to SJC.
-
-##### Where do I park?
-
-There is parking at the venue!
-
-##### Can we drink alcohol?
-
-Assuming you are of age, you are welcome to partake of fermented beverages. We do encourage moderation and require compliance with our [Code of Conduct](/coc) at all times.
-
-Please note that we are on federal lands, and federal law (*not* California law) takes precedence.
-
-##### What fun extra stuff do you have planned?
-
-Anyone who has attended in the past knows that we do some really fun stuff when we’re not working hard helping the nonprofits; hikes, meditation sessions, dance class, grown-up coloring books, and more. If you have an idea for something you’d like to do or lead, join us on slack! Don’t worry, we’ll still have plenty of time for karaoke, werewolf, and the other standbys!
-
-Our amazing [venue](/naturebridge) also has a campfire so we'll be able to do s'mores in the evenings! Last year Jennifer made marshmallow performance art. Bet you can't top that this year!
-
-##### What does my registration go toward?
-
-We are a break-even event.  Venue costs run about $350 per person -- and that's just for meals and lodging.  Snacks (like s'mores making!), earplugs, nametags, office supplies, lost key fines, and other minutia aren't included in that cost.  It turns out conferences require a lot of funds to put on!  Your registration fees go directly into funding the conference, and sponsorships (from organizations and individuals) help bring in our project partners and scholarship recipients.
-
-Every year we receive a number of amazing scholarship applicants who remain waitlisted until we have the funds to bring them in.
-If you'd like to help, [please consider making a donation to bring in our project partners and scholarship recipients](https://ti.to/codeforgood/rubybythebay)!
-
 ##### When do tickets go on sale?
 
 <!-- Very soon. We can't wait either! Let a friend know so you both can buy a ticket before we sell out. -->
-[Tickets are on sale now! buy yours today -- and buy one for a friend or scholarship recipient!](https://ti.to/codeforgood/rubybythebay) Last year was so fun. Tell all your #rubyfriends!
+[Tickets are on sale now! buy yours today](https://ti.to/codeforgood/rubybythebay) Last year was so fun. Tell all your #rubyfriends!
 
-##### Do I really HAVE to get there by 5pm on Friday?
+##### Do I really HAVE to be there by 5pm on Friday for kickoff?
```

Comment:
I took the liberty to change instead of removing this one. Let me know if a proper removal is warranted.